### PR TITLE
refactor(backend): inline NDB_VERSION

### PR DESCRIPTION
**Pull request by ak239**
As less environment variable we pass around as easier it to run
a little script to allow external node instances to connect to ndb.

drive-by: inspector.open to start inspector instead of passing
 --inspect flag.

### CODE REVIEW

File: lib/preload/ndb/preload.js [hunk 1]
```diff
@@ -25,14 +25,16 @@
     process.env.NDD_PPID = process.pid;
     if (!process.env.NDD_DATA)
       process.env.NDD_DATA = process.pid + '_ndbId';
-    process.versions['ndb'] = process.env.NDB_VERSION;
+    process.versions['ndb'] = '1.1.0';
+    const inspector = require('inspector');
```
Comment (eugeneo) [inline lib/preload/ndb/preload.js]: nit: There seem to be some unrelated changes. Consider splitting into several commits or amending the commit message.

Comment (ak239) [reply lib/preload/ndb/preload.js]: done.
